cypress/e2e/Class01.cy.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="Cypress" />
+
+describe('Validate EngageSphere website', ()=> {
+  beforeEach(()=> {
+    cy.visit('/')
+  })
+  context('Title', ()=> {
+    it('Validate title', ()=> {
+      cy.contains('h1','EngageSphere').should('be.visible')
+    })
+    it('Validate theme icon', ()=>{
+      cy.get('[id="theme-toggle-button"]').should('be.visible')
+    })   
+  })
+  context('Label', ()=>{
+    it('Validate default greting', ()=>{
+      cy.contains('Hi there!').should('be.visible')
+    })
+    it('Validate personal greting', ()=>{
+      cy.get('[data-testid="name"]').click().type('Caio')
+      cy.contains('Hi Caio').should('be.visible')
+    })
+  })
+  context('Modal customer details', ()=>{
+    it('Validate back button', ()=>{
+      cy.get('tbody > :nth-child(1) > :nth-child(1)').click()
+      cy.get('.customer-details').should('be.visible')
+      cy.contains('Back').click()
+      cy.get('[data-testid="table"]').should('be.visible')
+
+    })
+    it('Validate client adress', ()=> {
+      cy.get('tbody > :nth-child(1) > :nth-child(1)').click()
+      cy.get('.show-address-btn').click()
+      cy.get('.address-info').should('be.visible')
+    })
+
+    it('validate hide client adress', ()=> {
+      cy.get('tbody > :nth-child(1) > :nth-child(1)').click()
+      cy.get('.show-address-btn').click()
+      cy.get('.address-info').should('be.visible')
+      cy.get('.hide-address-btn').click()
+      cy.get('.show-address-btn').should('be.visible')
+    })
+    it('Validate if name field is disabled after open client details', ()=> {
+      cy.get('tbody > :nth-child(1) > :nth-child(1)').click()
+      cy.get('.show-address-btn').click()
```

Comment:
```suggestion
      cy.get('.show-address-btn').click()

```

É uma boa prática deixar uma linha em branco entre ações e resultados esperados, para facilmente identificar estas diferentes partes de um teste.

Reply:
O mesmo vale para outros lugares ao longo do seu código.